processors/base/writers/trial.py

```diff
@@ -30,8 +30,11 @@ def write_trial(conn, trial, source_id):
     create = False
     timestamp = datetime.datetime.utcnow()
 
-    # Get trial object
-    object = helpers.find_trial_by_identifiers(conn, identifiers=trial['identifiers'])
+    # Get trial object (first try to ignore source record for better dedup)
+    object = helpers.find_trial_by_identifiers(conn, trial['identifiers'],
```

Comment:
There should be no bad side effects here, but reusing reserved words like this can create some tricky bugs.